sources/packages/backend/apps/api/src/route-controllers/student/_tests_/e2e/student.aest.controller.updateProfileInformation.e2e-spec.ts

```diff
@@ -0,0 +1,176 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import {
+  AESTGroups,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAESTToken,
+} from "../../../../testHelpers";
+import {
+  E2EDataSources,
+  createE2EDataSources,
+  saveFakeStudent,
+} from "@sims/test-utils";
+import { IdentityProviders } from "@sims/sims-db";
+import * as faker from "faker";
+
+interface StudentProfileUpdateInfo {
+  givenNames: string;
+  lastName: string;
+  birthdate: string;
+  email: string;
+  noteDescription: string;
+}
+
+describe("StudentAESTController(e2e)-updateProfileInformation", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+  let studentProfileUpdateInfo: StudentProfileUpdateInfo;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+    studentProfileUpdateInfo = {
```

Comment:
Resuing the same profile information cannot lead to a unique key error on the user's table?